Sources/MapboxNavigation/NavigationViewController.swift

```diff
@@ -730,13 +730,21 @@ extension NavigationViewController: NavigationServiceDelegate {
         
         arrivalController?.showEndOfRouteIfNeeded(self,
                                                   advancesToNextLeg: advancesToNextLeg,
-                                                  completion: nil,
+                                                  completion: { [ weak self] _ in
+                                                    self?.frameDestinationArrival(for: service.router.location)
+                                                  },
```

Comment:
Do we really need this part? `frameDestinationArrival(for:)` is basically duplication of already existing logic, which updates user location puck on each `MapView` frame change. I've also tested that there is no difference with/without this part, so probably better to remove it. `CameraOptions` change looks good.

Reply:
Thanks for the suggestion, I'll remove this part. And should we also keep the zoom level after arrival same as 1.x ?